src/OrchardCore/OrchardCore.Media.Abstractions/MediaOptions.cs

```diff
@@ -67,6 +68,16 @@ public class MediaOptions
         /// </summary>
         public StaticFileOptions StaticFileOptions { get; set; }
 
+        /// <summary>
+        /// The maximum chunk size when uploading files in bytes. If 0 or null, no chunked upload is used. Defaults to 100MB.
+        /// </summary>
+        public int? MaxUploadChunkSize { get; set; }
```

Comment:
The defaults are set in https://github.com/Lombiq/OrchardCore/blob/issue/GOV-16/src/OrchardCore.Modules/OrchardCore.Media/Services/MediaOptionsConfiguration.cs but you are right, it can be an `int`.

Reply:
As I see [CulturePickerOptions](https://github.com/Lombiq/OrchardCore/blob/issue/GOV-16/src/OrchardCore.Modules/OrchardCore.ContentLocalization/CulturePickerOptions.cs) and [HealthChecksOptions](https://github.com/Lombiq/OrchardCore/blob/issue/GOV-16/src/OrchardCore.Modules/OrchardCore.HealthChecks/HealthChecksOptions.cs) the defaults are set on the options itself

Reply:
You are right, but in `OrchardCore.Media` an `IConfigureOptions<>` implementation is used to parse settings and set the defaults, so I'm just followed the existing solution.